sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -140,7 +141,7 @@ await using (var client = new EventHubClient(connectionString, scope.EventHubNam
                     Assert.That(properties, Is.Not.Null, "A set of properties should have been returned.");
                     Assert.That(properties.Path, Is.EqualTo(scope.EventHubName), "The property Event Hub name should match the scope.");
                     Assert.That(properties.PartitionIds.Length, Is.EqualTo(partitionCount), "The properties should have the requested number of partitions.");
-                    Assert.That(properties.CreatedAtUtc, Is.EqualTo(DateTime.UtcNow).Within(TimeSpan.FromSeconds(2)), "The Event Hub should have been created just about now.");
+                    Assert.That(properties.CreatedAtUtc, Is.EqualTo(DateTime.UtcNow).Within(TimeSpan.FromSeconds(5)), "The Event Hub should have been created just about now.");
```

Comment:
@jsquire I have increased the time span because it was too strict (tests were failing in my machine). Let me know if you disagree.

Reply:
We'll need to play with things a bit to make sure they're stable across environments.  Timing is always a delicate thing.

Reply:
It is likely a good idea to take the same approach here as you did with the partition properties and do a `[TestCase]` for each of the `TransportType` members.   That would satisfy the "can we connect and do a thing using all the available transports" twice over, which I think is as far as we need to go for proving that.